apps/is-prime/src/App.jsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import styled from "styled-components";
+import { isTriggered } from "@felvin-search/core";
+
+//------------Styled Components-------------
+// If you're unfamiliar with styled components
+// start here https://styled-components.com/docs/basics#getting-started
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+//=========================================
+
+// Your UI logic goes here.
+// `data` prop is exactly what is returned by queryToData.
+function Component({ data }) {
+  var matches = data.match(/(\d+)/);
+  var number = matches[0];
+  const isPrime=(num)=>{
+    for(let i=2;i<num;i++)
```

Comment:
Although checking like this isn't wrong per se, but it's slow for large numbers. This can make someone's site slow for a query like "is 688846502588399 a prime number". 

We can make some adjustments like running the loop only till the square root of the number, and skipping all the even numbers once it has passed 2 etc. Why don't you try these out and see the difference they make. You may find better ways on searching.

Reply:
True! 
The complexity of the code reduces to square root of the number.